src/main/res/values-de/strings.xml

```diff
@@ -31,15 +31,15 @@
     <string name="settings_theme">Desgin</string>
     <string name="settings_theme_light">Hell</string>
     <string name="settings_theme_dark">Dunkel</string>
-    <string name="settings_keep_on_title">Bildschirm angeschaltet lassen</string>
+    <string name="settings_keep_on_title">Bildschirm eingeschaltet lassen</string>
     <string name="settings_keep_on_summary">Bildschirm anlassen solange die App aktiv ist</string>
     <string name="settings_controls">Kontrolle</string>
     <string name="settings_sounds_title">Sound aktivieren</string>
-    <string name="settings_sounds_summary">Sounds abspielen</string>
+    <string name="settings_sounds_summary">Töne abspielen wenn Werte geändert werden</string>
     <string name="settings_vibration_title">Vibration aktivieren</string>
     <string name="settings_vibration_summary">Bei Änderungen vibrieren</string>
     <string name="settings_hard_title">Hardwaretasten benutzen</string>
-    <string name="settings_hard_summary">Wert mit Hardwaretasten verändern</string>
+    <string name="settings_hard_summary">Werde mit Lautstärkereglern verändern</string>
```

Comment:
It should be `Werte`

Reply:
This needs to be changed in Crowdin. That's where the changes come from.